### PR TITLE
feature(sct-runner): increase root disk size to 35 GB

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -605,17 +605,6 @@ def get_cluster_aws(self, loader_info, db_info, monitor_info):
...
         regions = self.params.get('region_name').split()
 
-        def get_root_device_name(ami_image_id):
-            for region in regions:
-                ec2 = boto3.resource('ec2', region)
-                image = ec2.Image(ami_image_id)
-                try:
-                    if image.root_device_name:
-                        return image.root_device_name
-                except (TypeError, ClientError):
```
Comment (enaydanov) [inline sdcm/tester.py:615]: Looks like it was the only place where `ClientError` was used. Need to remove unused import.

File: sdcm/utils/common.py [hunk 2]
```diff
@@ -1949,3 +1950,13 @@ def ec2_instance_wait_public_ip(instance):
...
+
+
+def ec2_ami_get_root_device_name(image_id, region):
+    ec2 = boto3.resource('ec2', region)
+    image = ec2.Image(image_id)
+    try:
+        if image.root_device_name:
+            return image.root_device_name
+    except (TypeError, ClientError):
+        raise AssertionError(f"Image '{image_id}' details not found in '{region}'")
```
Comment (enaydanov) [inline sdcm/utils/common.py:1962]: Exceptions will be chained. Do you expect this behavior? Otherwise, I can suggest to use `raise ... from None` syntax.

Comment (bentsi) [reply sdcm/utils/common.py:1962]: Yep, better to see why the Assertion happened

File: sdcm/tester.py (hunk 1 above)
Comment (bentsi) [reply sdcm/tester.py:615]: thanks, removed.

Comment (bentsi) [reply sdcm/tester.py:615]: nice catch!

**Pull request merged**